src/endpoints/account-memberships.js

```diff
@@ -30,6 +30,26 @@ class AccountMembershipsEndpoint extends BaseExtend {
     return this.call
   }
 
+  AllUnassignedAccountMembers(accountId, token = null) {
```

Comment:
Is there any reason to call them `AllUnassigned...`? If not I would prefer if we stay close to the api with naming and call it `UnassignedAccountMembers()`